src/transformers/utils/quantization_config.py

```diff
@@ -418,3 +422,8 @@ def post_init(self):
                     f"""dataset needs to be either a list of string or a value in
                     ['wikitext2','c4','c4-new','ptb','ptb-new'], but we found {self.dataset}"""
                 )
+        if not self.disable_exllamav2 and not self.disable_exllama:
+            logger.warning(
+                "You have activated exllama and exllamav2 backend. Setting `disable_exllama=True` and keeping `disable_exllamav2=False`"
+            )
+            self.disable_exllama = True
```

Comment:
Let's rather error out

Reply:
Depends on what we end up choosing above

Reply:
I've raised an error.